docs/src/main/sphinx/release/release-455.md

```diff
@@ -0,0 +1,60 @@
+# Release 455 (28 Aug 2024)
+
+## General
+
+* Add query starting time in `QueryStatistics` in all [](admin-event-listeners).
+  ({issue}`23113`)
+* Add JMX metrics for the bean
+  `trino.execution.executor.timesharing:name=TimeSharingTaskExecutor` replacing
+  metrics previously found in `trino.execution.executor:name=TaskExecutor`.
+  ({issue}`22914`)
+* Fix memory tracking issue for aggregations that could cause worker crashes
+  with out-of-memory errors. ({issue}`23098`)
+* Fix loading of custom `CatalogStore` implementations. Set `catalog.store` to
+  the name of the custom `CatalogStoreFactory`. `catalog-store.name` in
+  `etc/catalog-store.properties` is no longer required. ({issue}`22554`)
+* For file-based catalog store the relevant configuration now needs to be done
+  in `etc/catalog-store.properties` instead of `config.properties`.
+  ({issue}`23115`)
+
+## Delta Lake connector
+
+* Allow configuring endpoint for the native Azure filesystem. ({issue}`23071`)
+* Improve Glue connections with use exponential backoff strategy. ({issue}`23039`)
+
+## ClickHouse connector
+
+* Add support for creating tables with the `MergeTree` engine without the
+  `order_by` table property. ({issue}`23048`)
+
+## Hive connector
+
+* Allow configuring endpoint for the native Azure filesystem. ({issue}`23071`)
+* Improve Glue connections with use exponential backoff strategy. ({issue}`23039`)
+* Fix query failures due to "Multiple entries with same key" when reading
```

Comment:
@raunaqmorarka .. doesnt that apply to all object storage connectors that support Parquet?

Reply:
It was only for hive, as noted in the PR release notes